### DISCUSSION TITLE
Please sort the programs in the "Open with" menu alphabetically

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1301] [Suggestions]
Please sort the programs in the "Open with" menu alphabetically. It will improve the usability.

**Comment by rich2014** [comment]
@Andreas0602 

done.

download the snapshot and test it please.

**Comment by ghost** [comment]
I've tested it. It works very well. Also, the new entry "Other..." is very good. Thank you very much, @rich2014!
